README.md

```diff
@@ -60,7 +60,15 @@ We could retrieve this value from the repo settings through an API call but that
 
 Default: true
 
-The action will skip PRs that have failed checks.
+The action will skip PRs that have failed checks. Please note that if `allow_ongoing_checks` is not set to `true`, the action will skip PRs with ongoing checks. This may result in the failure to update PR branches when the action is triggered while checks for those PRs are still in progress.
+
+### `allow_ongoing_checks`
+
+**Optional**
+
+Default: false
+
+The action will consider PRs that have ongoing checks. This is useful when the action is triggered while checks for some otherwise qualified PRs are still in progress.
```

Comment:
This option only works if `require_passed_checks` is `true` (even though it's the default value). We can add a short comment here.

Reply:
Thanks